components/brave_new_tab_ui/components/default/gridSites/index.ts

```diff
@@ -5,7 +5,7 @@
 
 import styled, { css } from 'styled-components'
 
-export const TileTitle = styled('p')<{}>`
+export const TileTitle = styled('p') <{}>`
```

Comment:
Do we still need to undo these changes and keep them consistent?

Reply:
Sorry, I thought I fixed that :laughing: For some reason eslint isn't picking it up when I run `npm run eslint` and it doesn't fix them with `npm run eslint -- --fix` either. For now, I've manually fixed them for all the ts/tsx files I touched.

Reply:
thanks!